apps/transport/lib/jobs/consolidate_bnlc_job.ex

```diff
@@ -9,11 +9,20 @@ defmodule Transport.Jobs.ConsolidateBNLCJob do
   @schema_name "etalab/schema-lieux-covoiturage"
   @separator_key "csv_separator"
   @download_path_key "tmp_download_path"
+  # You may need to change this URL if you're working locally on the code.
+  # Maybe upload your own CSV to https://gist.github.com or return hardcoded
+  # slugs from `datagouv_dataset_slugs`
   @datasets_list_csv_url "https://raw.githubusercontent.com/etalab/transport-base-nationale-covoiturage/main/datasets.csv"
   @bnlc_github_url "https://raw.githubusercontent.com/etalab/transport-base-nationale-covoiturage/main/bnlc-.csv"
-  @bnlc_path "/tmp/bnlc.csv"
+  @bnlc_path System.tmp_dir!() |> Path.join("bnlc.csv")
```

Comment:
Ça ne va pas marcher en production ! Là c'est évalué à compile time, sur le builder, le répertoire n'existera dans la majorité des cas pas sur la machine de déploiement !

Désolé je t'ai induit en erreur, il faudrait avoir une méthode qui calcule au runtime.

Reply:
@AntoineAugusti pour préciser, par exemple sur ma machine ça crée des répertoires comme:

`"/var/folders/hz/7zpbnj551z5169215hq24gy40000gn/T/"`

Reply:
Yes, voir mon dernier message sur la discussion de la PR. Sur Mac il y a une variable d'env `TMPDIR` https://superuser.com/questions/581347/where-is-the-temp-folder-in-os-x et on n'a pas ça en prod, juste un fallback de `/tmp`